libkpod/container_data.go

```diff
@@ -176,3 +188,23 @@ func getBlankSpec() specs.Spec {
 		Windows:     &specs.Windows{},
 	}
 }
+
+// State copies the crio container state to ContainerState type for kpod
+func (c *ContainerServer) State(ctr *oci.Container) *ContainerState {
```

Comment:
Nit, but when I see a function with a name like State, I expect that when it's called it will return that object, in this case State.  Could you pick a verb of your liking and add it?  TransferCrioState, CopyCrioState, or some such?